syntax = "proto3";

package audio_enhancement;

service AudioEnhancer {
  rpc EnhanceAudio (EnhanceRequest) returns (EnhanceResponse);
}

message EnhanceRequest {
  string input_path = 1;
  string output_path = 2;
}

message EnhanceResponse {
  bool success = 1;
  string error = 2;
  string output_path = 3;
}
